test/integration/run_gadgetron_test.py

```diff
@@ -619,8 +619,9 @@ def stdout_compliance_action(cont, **state):
         files.append(os.path.join(args.test_folder, 'gadgetron.log.out'))
 
         for file in files:
-            if os.stat(file).st_size != 0:
-                raise RuntimeError(f"stdout is not empty as indicated by {file}")
+            if os.path.isfile(file): 
```

Comment:
How would it not be a file?

Reply:
the file does not exist at the beginning of test, if running "python run_gadgetron_test.py ...", then os.stat(file) leads to exception.